### DISCUSSION TITLE
For You says I follow the person making the release

### DISCUSSION THREAD

**Discussion by colejohnson66** [community/community#65762] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

As the title says, I don't follow this user:

![image](https://github.com/community/community/assets/11381599/d7255f06-6b21-4976-9a92-29c010e1eed3)

If we're going to have an algorithmic feed, at least don't be wrong.

**Comment by colejohnson66** [comment]
I follow `apple`, but not `apple/foundationdb`

**Comment by colejohnson66** [comment]
Another case: I follow JetBrains, but not this person or the repo:

![image](https://github.com/community/community/assets/11381599/d3c6cace-0799-4cb0-a721-a64e93a54307)

**Reply by colejohnson66** [reply]
It looks like it's saying I follow the person who posted the release because a nanoframework release says I follow nfbot:

![image](https://github.com/community/community/assets/11381599/a85eba67-490d-497e-ae3a-8e83e9e4c923)

**Comment by joe-bro-1** [comment]
i have the same problem!
as far as i know there's no way to fix it because you cannot unfollow someone if you aren't following them🤯!
